src/libs/OptionsListUtils.js

```diff
@@ -83,12 +83,13 @@ function createOption(personalDetailList, report, draftComments, activeReportID,
         && lodashGet(draftComments, `${ONYXKEYS.COLLECTION.REPORT_DRAFT_COMMENT}${report.reportID}`, '').length > 0;
 
     const lastActorDetails = report ? _.find(personalDetailList, {login: report.lastActorEmail}) : null;
-    const lastMessageText = report
+    let lastMessageText = report
         ? (hasMultipleParticipants && lastActorDetails
             ? `${lastActorDetails.displayName}: `
             : '')
         + report.lastMessageText
         : '';
+    lastMessageText = _.unescape(lastMessageText);
```

Comment:
Let's just escape the `report.lastMessageText` since the `displayName` is an email that wouldn't have any HTML entities in it. That way we can also keep `lastMessageText` as a `const`

Reply:
Resolved according to your feedback